platform/linuxbsd/display_server_x11.cpp

```diff
@@ -3806,7 +3806,9 @@ DisplayServerX11::~DisplayServerX11() {
 	for (Map<WindowID, WindowData>::Element *E = windows.front(); E; E = E->next()) {
 #ifdef VULKAN_ENABLED
 		if (rendering_driver == "vulkan") {
-			context_vulkan->window_destroy(E->key());
+			if (context_vulkan) {
```

Comment:
I'd make this an `ERR_FAIL_NULL`, if we don't have a context to destroy the windows, but we still have windows to destroy, we're going to leak (or crash).